### ISSUE TITLE
Try SciPy L-BFGS optimizer

### ISSUE THREAD

**Issue by DerkNiessink** [DerkNiessink/PEPSFlow#81]
Apparently the L-BFGS optimizer should not have a learning rate parameter. For some reason this is the case in PyTorch and not in SciPy.

**Comment by DerkNiessink** [comment]
Will close this for now; LBFGS pytorch implementation works fine.
